src/oss-find-squats-lib/Mutators/DoubleHitMutator.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.CST.OpenSource.FindSquats.Mutators
+{
+    /// <summary>
+    /// Generates mutations for if a nearby character was double pressed.
+    /// </summary>
+    public class DoubleHitMutator : IMutator
+    {
+        public MutatorType Kind { get; } = MutatorType.DoubleHit;
+
+        public IEnumerable<Mutation> Generate(string arg)
+        {
+            for (int i = 0; i < arg.Length; i++)
+            {
+                List<char>? n = QwertyKeyboardHelper.GetNeighboringCharacters(arg[i]).ToList();
+
+                foreach (char c in n)
+                {
+                    yield return new Mutation(
+                        mutated: string.Concat(arg[..i], c, arg[i..]),
+                        original: arg,
+                        mutator: Kind,
+                        reason: $"Double hit character: {c}");
+                    yield return new Mutation(
+                        mutated: string.Concat(arg[..(i + 1)], c, arg[(i + 1)..]),
+                        original: arg,
+                        mutator: Kind,
+                        reason: $"Double hit character: {c}");
```

Comment:
Looking over this now, I'm trying to figure out what the difference is?
Isn't this just adding the doubled character before, and then another time, after? I think that would be the same thing right?
Like loda **s** sh, and lodas **s** h? It's the same string but the s was duplicated once before the s, and another mutation with the duplicated s after the s.

Reply:
I believe I copied this from your PR and also am unclear on what the second one does. Can remove.

Reply:
I copied it from the original `Generative.cs` file 😅

Reply:
I see that this is the case in main as well, but I still do not remember why we did this.